router-bridge/src/js.rs

```diff
@@ -61,11 +64,17 @@ impl Js {
                 .expect("unable to evaluate service list in JavaScript runtime");
         }
 
-        runtime.execute_script(name, source).unwrap_or_else(|e| {
-            panic!(
+        // We are sending the error through the channel already
+        let _ = runtime.execute_script(name, source).map_err(|e| {
+            let message = format!(
                 "unable to invoke {} in JavaScript runtime \n error: \n {:?}",
                 source, e
-            )
+            );
+
+            tx.send(Err(Error::JSRuntimeError(message)))
+                .expect("channel must be open");
+
+            deno_core::anyhow::Error::from(e)
```

Comment:
I think you can just return `e`... I'm surprised clippy didn't complain about that.

Reply:
possibly because we're not linting in this repo 🙈 lemme check real quick :D